Carta.py

```diff
@@ -548,6 +661,10 @@ def process(self):
                     self.decideYourGrabPhaseStatus()
                     self.opponentRespondsInGrabPhase()
                     self.decideWinner()
+            if (self.phase is Phase.OPPONENT_TRANSFER):
+                self.opponentTransfer()
+            if (self.phase is Phase.YOUR_TRANSFER):
+                self.renderDoneButton()
```

Comment:
Need to do these

1. self.dialogs print "Give 1 card to opponent" or "Give 2 cards to opponent"
2. When your field has the right number of yellow cards, press Done button, these cards show up in opponent's field randomly, then change phase to YOUR_SET_UP